core/src/main/scala/kafka/server/KafkaHealthcheck.scala

```diff
@@ -117,18 +96,18 @@ class KafkaHealthcheck(brokerId: Int,
 
     @throws[Exception]
     override def handleNewSession() {
-      info("re-registering broker info in ZK for broker " + brokerId)
-      register()
-      info("done re-registering broker")
-      info("Subscribing to %s path to watch for new topics".format(ZkUtils.BrokerTopicsPath))
+      //info("re-registering broker info in ZK for broker " + brokerId)
```

Comment:
I think commented out code should be removed.

Reply:
Yes, the plan is to do that in KAFKA-6320.